gyp/pylib/gyp/xcode_emulation.py

```diff
@@ -1541,6 +1541,13 @@ def CLTVersion():
         except GypError:
             continue
 
+    regex = re.compile('Command Line Tools for Xcode\s+(?P<version>\S+)')
+    try:
+        output = GetStdout(['/usr/sbin/softwareupdate', '--history'])
+        return re.search(regex, output).groupdict()['version']
+    except:
```

Comment:
Please, no bare exceptions https://realpython.com/the-most-diabolical-python-antipattern/

Reply:
Obviously I'm not a Python wizard. :) Fixed.